robotoff/prediction/ocr/category_from_AOC.py

```diff
@@ -76,7 +80,14 @@ def category_taxonomisation(lang, match) -> Optional[str]:
 }
 
 
-def find_category_from_AOC(content: Union[OCRResult, str]) -> List[Prediction]:
+def find_category(content: Union[OCRResult, str]) -> List[Prediction]:
+    """This function returns a prediction of the category of the product
+    by detecting an AOP syntax which allows an easy category
+    prediction with REGEX. 
+    For now we are extracting categories via REGEX 
+    only thanks to an AOP syntax but we may find in the future 
+    other ways to get sure prediction of categories.
+    """
```

Comment:
I won't focus on AOP here. Your function could work for other categories as well.

That's also why I proposed you to rename the file to category.py (and you should do it ;-))

The only thing which is specific to AOP is the Regexp you defined and their processing function.
So we can keep it generic, one might as well add another regexp and processing function to add categories about honey type…

Reply:
Indeed ! I should have removed the mention of AOP in the docstring. Regarding the file name, that's weird because I have it under the name category.py in my local repository.... I'll commit it again then !